frontend/src/components/projects/ProjectEditContent.js

```diff
@@ -169,11 +166,14 @@ function deserialize(values) {
 function serialize(values) {
   const newValues = {...values}
 
+  if (newValues.status === false) newValues.status = "Closed"
```

Comment:
I think we should stick to putting code on a new line after an if or an else ...